src/integrationTest/java/org/opensearch/security/Song.java

```diff
@@ -26,19 +30,34 @@ class Song {
 	static final String ARTIST_NO = "No!";
 	static final String TITLE_POISON = "Poison";
 
+	public static final String ARTIST_YES = "yes";
+
+	public static final String TITLE_AFFIRMATIVE = "Affirmative";
+
+	public static final String ARTIST_UNKNOWN = "unknown";
+	public static final String TITLE_CONFIDENTIAL = "confidential";
+
 	public static final String LYRICS_1 = "Very deep subject";
 	public static final String LYRICS_2 = "Once upon a time";
 	public static final String LYRICS_3 = "giant nonsense";
 	public static final String LYRICS_4 = "Much too much";
+	public static final String LYRICS_5 = "Little to little";
+	public static final String LYRICS_6 = "confidential secret classified";
+
+	static final String GENRE_ROCK = "rock";
+	static final String GENRE_JAZZ = "jazz";
+	static final String GENRE_BLUES = "blues";
 
 	static final String QUERY_TITLE_NEXT_SONG = FIELD_TITLE + ":" + "\"" + TITLE_NEXT_SONG + "\"";
 	static final String QUERY_TITLE_POISON = FIELD_TITLE + ":" + TITLE_POISON;
 	static final String QUERY_TITLE_MAGNUM_OPUS = FIELD_TITLE + ":" + TITLE_MAGNUM_OPUS;
 
 	static final Object[][] SONGS = {
```

Comment:
This is complex and a small error could blow up many tests for an unknown reason, what do you think about introducing a type representing a song scheme to ensure consistent construction?

Reply:
You are right. This data structure is getting more and more complex. I will create a DTO class which contains fields like artist, title, etc. and will serialize the class to JSON with a Jackson library.

Reply:
Finally I manage to do this without Jackson.